dev/com.ibm.ws.concurrent.mp.1.0/bnd.bnd

```diff
@@ -14,42 +14,52 @@ bVersion=1.0
 javac.source: 1.8
 javac.target: 1.8
 
-Bundle-Name: MicroProfile Context Propagation 1.0 Concurrency function
+Bundle-Name: MicroProfile Context Propagation 1.0 implementation
 Bundle-SymbolicName: com.ibm.ws.concurrent.mp.1.0
-Bundle-Description: Concurrency function used by MicroProfile Context Propagation 1.0 implementation; version=${bVersion}
+Bundle-Description: Implementation of MicroProfile Context Propagation 1.0; version=${bVersion}
 
 Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
 
 WS-TraceGroup: concurrent
 
 Import-Package:\
   javax.inject;resolution:=optional,\
+  !org.eclipse.microprofile.config,\
+  !org.eclipse.microprofile.config.spi,\
   *
 
-Export-Package: \
-  com.ibm.ws.concurrent.mp.spi.*
+DynamicImport-Package:\
+  org.eclipse.microprofile.config,\
+  org.eclipse.microprofile.config.spi
 
 Private-Package:\
-  com.ibm.ws.concurrent.mp.*
+  com.ibm.ws.concurrent.mp.*,\
+  com.ibm.ws.microprofile.context.*,\
+  com.ibm.ws.microprofile.contextpropagation
 
-instrument.classesExcludes: com/ibm/ws/concurrent/mp/resources/*.class
+instrument.classesExcludes: com/ibm/ws/microprofile/context/resources/*.class
 
 -dsannotations: \
-  com.ibm.ws.concurrent.mp.ContextServiceImpl,\
-  com.ibm.ws.concurrent.mp.ManagedExecutorImpl,\
-  com.ibm.ws.concurrent.mp.ManagedScheduledExecutorImpl,\
-  com.ibm.ws.concurrent.mp.spi.CompletionStageFactory
+  com.ibm.ws.concurrent.mp.service.ContextServiceImpl,\
+  com.ibm.ws.concurrent.mp.service.ManagedExecutorImpl,\
+  com.ibm.ws.concurrent.mp.service.ManagedScheduledExecutorImpl,\
+  com.ibm.ws.microprofile.contextpropagation.ContextManagerProviderImpl,\
+  com.ibm.ws.microprofile.contextpropagation.MicroProfileClearedContextProvider,\
+  com.ibm.ws.microprofile.contextpropagation.MPConfigAccessorImpl
 
 -buildpath: \
   com.ibm.websphere.appserver.spi.logging,\
   com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
   com.ibm.websphere.org.eclipse.microprofile.contextpropagation.1.0,\
+  com.ibm.websphere.org.eclipse.microprofile.config.1.4;version=latest,\
```

Comment:
I wasn't aware that MP Context Propagation had a dependency on MP Config? Which feature actually pulls it in and at what version? I'm happy if the feature pulls in 1.4 by default but if it tolerates lower versions then this buildpath should specify the lowest version tolerated.

Reply:
Updated.  I had put it back to what it was before.  I changed it to 1.1 now.